app/lib/sharezone_plus/page/sharezone_plus_page_bloc.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2023 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'dart:async';
+
+import 'package:bloc_base/bloc_base.dart';
+import 'package:rxdart/subjects.dart';
+import 'package:sharezone/sharezone_plus/page/sharezone_plus_page_view.dart';
+import 'package:sharezone/sharezone_plus/subscription_service/purchase_service.dart';
+import 'package:sharezone/sharezone_plus/subscription_service/revenue_cat_sharezone_plus_service.dart';
+import 'package:sharezone/sharezone_plus/subscription_service/subscription_service.dart';
+
+/// A fallback price if the price cannot be fetched from the backend.
+///
+/// On macOS the price is not fetched from the backend because RevenueCat does
+/// not support macOS.
+const kFallbackPrice = '4,99 €';
+
+class SharezonePlusPageBloc extends BlocBase {
+  final BehaviorSubject<SharezonePlusPageView> _view =
+      BehaviorSubject<SharezonePlusPageView>.seeded(
+          SharezonePlusPageView.empty());
+  Stream<SharezonePlusPageView> get view => _view;
+
```

Comment:
Side note: What kind of state management do we want to use in the future? I mean wouldn't e.g. `Provider` be simpler to use instead of using bloc? (So e.g. `class SharezonePlusPageController extends ChangeNotifier`).  
Idk, I don't really see a benefit by using (DIY-)Blocs anymore, but I'm also not quite sure how we want to proceed in the future
(I mean we don't even have all the same state management everywhere, sometimes X might be better than Y and vice versa)

Reply:
Yes, I wasn't also sure about the stage management. I changed the `SharezonePlusPageBloc` to `SharezonePlusPageController` 👍  I also removed the `SharezonePlusPageView` because not sure if it's really worth now.